rebar.config

```diff
@@ -41,7 +41,7 @@
         {sha3, {git, "https://github.com/szktty/erlang-sha3", {ref, "dbdfd12"}}},
         {sext, {git, "https://github.com/uwiger/sext.git", {ref, "615eebc"}}},
         {idna, {git, "https://github.com/aeternity/erlang-idna", {tag, "6.0.0"}}},
-        {aecuckooprebuilt, {aecuckooprebuilt_app_with_priv_from_git, {git, "https://github.com/aeternity/cuckoo-prebuilt.git", {ref, "54f098d45a25490cd22bc80c87c7dcc937d53382"}}}}
+        {aecuckooprebuilt, {aecuckooprebuilt_app_with_priv_from_git, {git, "https://github.com/aeternity/cuckoo-prebuilt.git", {ref, "90afb699dc9cc41d033a7c8551179d32b3bd569d"}}}}
```

Comment:
Using artefact from [CI](https://circleci.com/gh/aeternity/epoch/48689#artifacts/containers/0) on this commit:
```
$ tar tf ~/Downloads/epoch-932ca2c09dd14bf5e48bba666ea1c64bc072b467-ubuntu-x86_64.tar.gz  | grep prebuilt
lib/aecuckooprebuilt-0.1.0/priv/cuda29
lib/aecuckooprebuilt-0.1.0/priv/lcuda29
lib/aecuckooprebuilt-0.1.0/ebin/aecuckooprebuilt.app
```